hpOneView/resources/facilities/power_devices.py

```diff
@@ -47,14 +47,227 @@ def __init__(self, con):
         self._connection = con
         self._client = ResourceClient(con, self.URI)
 
-    def get_utilization(self, id, fields=None, filter=None, refresh=False, view=None):
+    def get_all(self, start=0, count=-1, filter='', query='', sort=''):
+        """
+        Gets a set of power delivery device resources according to the specified parameters. Filters can be used to get
+        a specific set of power delivery devices. With no filters specified, the API returns a potentially paginated
+        list of all the power delivery device resources subject to start/count/sort parameters.
+
+        Args:
+            start:
+                The first item to return, using 0-based indexing.
+                If not specified, the default is 0 - start with the first available item.
+            count:
+                The number of resources to return. A count of -1 requests all the items.
+                The actual number of items in the response may differ from the requested
+                count if the sum of start and count exceed the total number of items, or
+                if returning the requested number of items would take too long.
+            filter:
+                A general filter/query string to narrow the list of items returned. The
+                default is no filter - all resources are returned.
+            query:
+                 A general query string to narrow the list of resources returned. The default
+                 is no query - all resources are returned.
+            sort:
+                The sort order of the returned data set. By default, the sort order is based
+                on create time, with the oldest entry first.
+
+        Returns: list of power devices
+        """
+        return self._client.get_all(start, count, filter=filter, sort=sort, query=query)
+
+    def get(self, id_or_uri):
+        """
+        Gets a single power delivery device resource based upon its uri or id.
+        Args:
+            id_or_uri:
+                Could be either the power device id or the uri
+
+        Returns:
+            dict: The power device
+        """
+        return self._client.get(id_or_uri)
+
+    def add(self, information, timeout=-1):
+        """
+        Adds a power delivery device resource based upon the attributes specified. Use this method to create a
+        representation of power delivery devices that provide power to other resources but cannot otherwise be
+        discovered by the management appliance. All attributes without default values must be specified in the POST
+        body. The response contains the power delivery device resource as added to the appliance with default and
```

Comment:
I guess the `POST body` in this comment is out of context. You could replace it by `information argument` or something like that.